gui/src/installer/view.rs

```diff
@@ -1088,12 +1091,107 @@ pub fn help_backup<'a>() -> Element<'a, Message> {
     text(prompt::BACKUP_DESCRIPTOR_HELP).small().into()
 }
 
-pub fn define_bitcoin<'a>(
+pub fn define_bitcoin_node<'a>(
     progress: (usize, usize),
+    available_node_types: impl Iterator<Item = NodeType>,
+    selected_node_type: &NodeType,
```

Comment:
`NodeType` implement `Copy`
```suggestion
    selected_node_type: NodeType,
```